module/VuFind/src/VuFind/XSLT/Import/VuFind.php

```diff
@@ -64,14 +64,14 @@ public static function setServiceLocator($serviceLocator)
     }
 
     /**
-     * Get the change tracker table object.
+     * Get the change tracker service object.
      *
-     * @return \VuFind\Db\Table\ChangeTracker
+     * @return \VuFind\Db\Service\ChangeTrackerService
```

Comment:
There are two calls to `getChangeTracker()` in this file which return a `$row`... and then subsequent code has not been adjusted to account for the changed return values.